### ISSUE TITLE
Incorporating dialogue data into classifier

### ISSUE THREAD

**Issue by egermani** [got-show/general#1]
I was looking at your features for the machine learning and thought it might help to have some info that the wikis can't provide. I did a dialogue analysis a little while ago, and perhaps the [lines spoken](http://atseajournal.com/asoiaf/resources/lines_per_char_table.html) / [words spoken](http://atseajournal.com/asoiaf/resources/words_per_char_table.html) per character may improve your classifier's performance.

**Comment by sacdallago** [comment]
Great data @egermani !

**Comment by sacdallago** [comment]
@got-show/gotplod @got-show/gotarffplod For the next iteration :)

**Comment by TatyanaGoldberg** [comment]
Indeed, we are very interested in seeing whether the character that GRRM LIKES to write about is on the top of the next-to-die list! Thanks @egermani for the idea!
